packages/shared/src/variants/baduk.ts

```diff
@@ -196,7 +217,10 @@ export class Baduk extends AbstractGame<BadukConfig, BadukState> {
 }
 
 /** Returns true if the group containing (x, y) has at least one liberty. */
-export function groupHasLiberties(group: CoordinateLike[], board: Grid<Color>) {
+export function groupHasLiberties(
+  group: CoordinateLike[],
+  board: Fillable<CoordinateLike, Color>,
```

Comment:
Should this be `BadukBoard<Color>` instead of `Fillable<...>`?

Reply:
changed type nominally

Reply:
changed type